project/src/main/webapp/discovery-style.css

```diff
@@ -0,0 +1,82 @@
+.header{
+  padding: 30px;
+  text-align: center;
+  background: white;
+}
+
+.subheader {
+  padding: 50px;
+  text-align: center;
+}
+
+.row:after {
+  content: "";
+  display: table;
+  clear: both;
+}
+
+.content-container {
+    text-align: center;
+}
+
+.profile-header {
+  padding: 20px;
+  margin-top: 20px;
+}
+
+.profile-header__left-column {
+  float: left;
```

Comment:
I recommending looking into flexbox (https://css-tricks.com/snippets/css/a-guide-to-flexbox/) in place of floats --- I think it's a personal preference, but a lot of people consider them much easier to work with.

Reply:
Ok cool, thanks!